integrationtest/performance/orm/src/test/resources/arquillian.xml

```diff
@@ -22,7 +22,11 @@
     <container qualifier="wildfly" default="true">
         <configuration>
             <property name="jbossHome">${project.build.directory}/wildfly-${wildflyVersion}</property>
-            <property name="javaVmArguments">-Xmx512m -Djava.net.preferIPv4Stack=true -Djgroups.bind_addr=127.0.0.1</property>
+            <property name="javaVmArguments">
+                -Xmx512m -Djava.net.preferIPv4Stack=true -Djgroups.bind_addr=127.0.0.1
+                -Dscenario=${scenario}
+                -Dorg.hibernate.search.enable_performance_tests=${org.hibernate.search.enable_performance_tests}
+            </property>
```

Comment:
Are you sue that's necessary? Adding them won't hurt but I'm wondering, as while WildFly is run in a separate JVM the test execution driver is not: the test methods are invoked over a remote protocol.

Reply:
Yes I'm sure. The properties are read from our own test classes, executed inside WildFly, namely in `TestContext` (when initializing `org.hibernate.search.test.performance.scenario.TestContext#performanceEnabled`) and `org.hibernate.search.test.performance.scenario.TestScenarioFactory#create`.